README.md

```diff
@@ -19,22 +19,14 @@ In separate Terminal tabs:
 In `client/.env` :
 ```
 NEXT_API_BASE_URL=http://localhost:8531/
+NEXTAUTH_URL_INTERNAL=http://localhost:8531/ # internal url of your a12n-server for development purposes
```

Comment:
So I think next would run off a different port than a12n-server. The intended design is (probably) that you have 2 separate endpoints like:

    http://localhost:8080 <- next app
    http://localhost:8531 <- a12n
    
In production this would then become something like

    http://app.example.org/
    http://auth.example.org/

Reply:
So that makes me think that the environment variables here are maybe a misnomer.

Reply:
The next port doesnt have to be configured as any react lib likes to default hog `localhost:3000` 😆 

However, I realize`NEXTAUTH_URL_INTERNAL` is only used in <= next-auth@v4, and is not mentioned in the v5 docs, so might as well get rid of it and refer to `AUTH_URL`... 
According to authjs they want to do away with prefixing `NEXT_`, which, would be pretty funny if people want to deploy on vercel I think 😆  https://authjs.dev/getting-started/migrating-to-v5#environment-variables